spec/features/user_can_update_length_of_rental_while_status_is_out_spec.rb

```diff
@@ -14,6 +14,10 @@
       visit dashboard_path(user_one)
       fill_in 'days-field', with: '2'
       click_button "Submit"
+      sleep(2)
+      click_on "Agree"
+
+      expect(page).to have_content('$56')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.